en/getting-started.md

```diff
@@ -1,10 +1,104 @@
+4.  Select all three of them and click `Open`, which will close this second dialog window.
+5.  Now that they all show up in the list in the Preferences dialog, make sure that they all are checked and close it .
+6.  If the Libraries have been added successfully, they will all show up in the `Symbols` section of the `Insert` menu in Sketch.
+
+### Creating your First Indigo Design Artboard
+
+Now that you have added the Indigo Design Libraries, you are all set to start crafting amazing experiences. Our first design will libraries will be for a simple registration screen with a Navbar, and a beautiful image for background on top of which we will design a simple form consisting of a little bit of Text, a few Inputs and a Button to submit the user details. So let's find out what a productivity boost the Indigo Design system brings to the table.
+
+1.  Open up the Sketch app on your Mac and create a new file. Press the `A` key and from the right panel select an artboard size from the panel on the right. In this example we will be using an `iPhone 8` in portrait orientation, which will give us a 375 by 667 artboard to design for. If you don't see this preset arboard, choose `Apple Devices` from the dropdown at the top of the panel.
+
+2.  From the insert menu, select `Indigo-Components`, then `Navigation` and `Navbar` positioning the Navbar at the top of the artboard and stretching it to take up the full witdth available. With the Navbar selected, fix its height and fix it to the left, top, and right from the resizing section in the right panel. Looking at the overrides section below we will change the `Type` to `~Title` and type "Sign Up" for the `Text` override below. Now let's drag to the artboard an image, which we will use as a background and make sure that it is positioned below the Navbar in the layers panel on the left, which should look like this:
+
+| Layer                          | Description                                  |
+| ------------------------------ | -------------------------------------------- |
+| iPhone 8                       | The artboard                                 |
+| &nbsp;&nbsp; Navigation/Navbar | The Navbar that we have inserted             |
+| &nbsp;&nbsp; Image             | The image, which we will use as a background |
+
+  <div class="divider--half"></div>
+![](../images/getting-started1.png)
+  <div class="divider--half"></div>
+
+3.  From the insert menu, select `Indigo-Styling`, then `Shadows/Rect` and `Elevate 18` positioning it above the background, but below the Navbar. Do the same to inser a `Colors/white` rectangle and resize and position it so that its borders are aligned with all the borders of the elevation. Now select both layers and size them to 280 by 400, putting them in the exact center of the artboard to serve as a surface on top of which our form will be laid out.
```

Comment:
Do the same to insert a `Colors/white` rectangle and resize and position it so that its borders are aligned with all the borders of the elevation.